co2eq/car/index.test.js

```diff
@@ -0,0 +1,107 @@
+import moment from 'moment';
+import {
+  TRANSPORTATION_MODE_CAR,
+  ACTIVITY_TYPE_TRANSPORTATION,
+  EUROCARSEGMENTS,
+  ENGINETYPES,
+} from '../../definitions';
+import cars from './cars.json';
+import {
+  modelCanRun,
+  carbonEmissions,
+  modelName,
+  modelVersion,
+  explanation,
+  modelCanRunVersion,
+} from './index';
+
+test(`model has valid API`, () => {
```

Comment:
nice! We should almost have a test that imports all models and verifies it for all of them?

Reply:
That's a very good point! I've updated this PR with one now and streamlined the structure of models, so we can easily add more tests :)